m4/libphonenumber.m4

```diff
@@ -0,0 +1,24 @@
+# https://github.com/googlei18n/libphonenumber
+AC_DEFUN([LIBPHONENUMBER_INIT],[
+  AC_SUBST(HAS_LIBPHONENUMBER) 
```

Comment:
libphonenumber is the 3rd party lib that handles the phone number validation/parsing